src/handlers.rs

```diff
@@ -673,3 +675,41 @@ fn handle_timer_clear(
   remove_timer(d, msg.id());
   Ok(null_buf())
 }
+
+fn handle_chmod_sync (
+  d: *const DenoC,
+  base: msg::Base,
+  _builder: &mut FlatBufferBuilder,
+) -> HandlerResult {
+  let msg = base.msg_as_chmod_sync().unwrap();
+  let path = msg.path().unwrap();
+  let mode = msg.mode().unwrap();
+  let deno = from_c(d);
+  debug!("handle_chmod_sync {} {}", path, mode);
+  if !deno.flags.allow_write {
+    let err = std::io::Error::new(
+      std::io::ErrorKind::PermissionDenied,
+      "allow_write is off.",
+    );
+    return Err(err.into());
+  }
+  let mut final_cmd = "chmod ".to_owned();
+  final_cmd.push_str(mode);
+  final_cmd.push_str(" ");
+  final_cmd.push_str(path);
+  println!("{}", final_cmd);
+  if cfg!(target_os = "windows") {
+    // Todo
+    Command::new("cmd")
+            .args(&["/C", "Yet to make for windows"])
+            .output()
+            .expect("failed to execute process")
+  } else {
+    Command::new("sh")
+            .arg("-c")
+            .arg(final_cmd.to_string())
+            .output()
+            .expect("failed to execute process")
```

Comment:
Starting a new process to do `chmod` is much too much overhead. Please call the syscall directly.